.travis.yml

```diff
@@ -19,8 +19,10 @@ before_script:
   - sh -e /etc/init.d/xvfb start
   - npm install -g npm
   - npm install
+env:
+  - MIX_ENV=test
 script:
-  - "mix do local.hex --force, deps.get && mix test --include pandoc"
+  - "mix do local.hex --force, deps.get, test --include pandoc, coveralls.travis"
```

Comment:
But I already exported the `MIX_ENV=test` via `.travis.yml`, I don't see any complain [here](https://travis-ci.org/elixir-lang/ex_doc/builds/80292616).